pkg/checks/health/metrics.go

```diff
@@ -0,0 +1,41 @@
+package health
+
+import (
+	"github.com/caas-team/sparrow/pkg/checks"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+// metrics contains the metric collectors for the Health check
+type metrics struct {
+	*prometheus.GaugeVec
+}
+
+// newMetrics initializes metric collectors of the health check
+func newMetrics() metrics {
+	return metrics{
+		GaugeVec: prometheus.NewGaugeVec(
+			prometheus.GaugeOpts{
+				Name: "sparrow_health_up",
+				Help: "Health of targets",
+			},
+			[]string{
+				"target",
+			},
+		),
+	}
+}
+
+// GetMetricCollectors returns all metric collectors of check
+func (h *Health) GetMetricCollectors() []prometheus.Collector {
+	return []prometheus.Collector{
+		h.metrics,
+	}
+}
+
+// Remove removes a metric with a specific label
+func (m metrics) Remove(label string) error {
```

Comment:
Is there a reason why you've chosen not to use a pointer receiver here or was this just unintended?

Reply:
Unintended. Should be a pointer, just for consistency's sake.